tests/api/test_all_stories.py

```diff
@@ -95,9 +99,9 @@
                 }
             ],
             'meta':{
+                'requested_node_index': 0,
```

Comment:
This change seems unnecessary.

Reply:
I put a pprint again and changed the expected output. Maybe that changed the order or something